<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mazeSizeBox">
     <item>
      <widget class="QLabel" name="mazeSizeLabel1">
       <property name="text">
        <string>请输入迷宫大小：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mazeSizeLabel2">
       <property name="text">
        <string>注意：奇数，小于xxx</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mazeSizeLine"/>
     </item>
     <item>
      <layout class="QGridLayout" name="numberBox_2">
       <item row="3" column="1">
        <widget class="QPushButton" name="mazeSize8">
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="mazeSize2">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="mazeSize7">
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="mazeSize5">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="mazeSizeBack">
         <property name="text">
          <string>⬅</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="mazeSize1">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="mazeSize6">
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="mazeSize3">
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="mazeSizeSure">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="mazeSize9">
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="mazeSize4">
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="mazeSize0">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="mazeTipLabel1">
       <property name="text">
        <string>     当前迷宫阶数为：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="mazeBegin">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始游戏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mazeAuto">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>自动寻路</string>
    </property>
   </widget>
   <widget class="rescaleImg" name="mazeGraphicsView">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>651</width>
      <height>601</height>
     </rect>
    </property>
    <property name="dragMode">
     <enum>QGraphicsView::DragMode::ScrollHandDrag</enum>
    </property>
   </widget>
   <widget class="QLabel" name="TimeCNT">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="actionGame"/>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>模式</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>无尽</string>
     </property>
     <addaction name="mode11"/>
     <addaction name="mode12"/>
    </widget>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>计时</string>
     </property>
     <addaction name="mode21"/>
     <addaction name="mode22"/>
    </widget>
    <addaction name="menu_3"/>
    <addaction name="menu_4"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actionGame">
   <property name="text">
    <string>Game</string>
   </property>
   <property name="shortcut">
    <string>Alt+G</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>Alt+H</string>
   </property>
  </action>
  <action name="mode11">
   <property name="text">
    <string>普通</string>
   </property>
  </action>
  <action name="mode12">
   <property name="text">
    <string>内嵌</string>
   </property>
  </action>
  <action name="mode21">
   <property name="text">
    <string>普通</string>
   </property>
  </action>
  <action name="mode22">
   <property name="text">
    <string>内嵌</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>rescaleImg</class>
   <extends>QGraphicsView</extends>
   <header>rescaleimg.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
